### PR TITLE
Pipes Milestone

### PR DESCRIPTION
**Pull request by DanBotero**
The Pipes Milestone done.

### CODE REVIEW

**Comment by DanBotero** [comment]
Pipes!
screen shot 2017-05-05 at 3 35 10 pm

File: src/app/demo/pipes/pipes.component.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import {Component} from '@angular/core';
+import {displayAngularComponent} from '../../exercise/helpers/helpers';
+// import { ng2tsConfig } from '../../../../ng2ts/ng2ts';
```
Comment (prestonvanloon) [inline src/app/demo/pipes/pipes.component.ts]: Delete this comment

File: src/app/demo/pipes/pipes.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(PipesComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
```
Comment (prestonvanloon) [inline src/app/demo/pipes/pipes.component.spec.ts]: This seems to be broken with travis. You'll need to add the component dependencies in your test module.

Comment (DanBotero) [inline src/app/demo/pipes/pipes.component.spec.ts]: Preston thanks!!  working on it now.

**Pull request merged**